README.md

```diff
@@ -33,6 +33,8 @@ KubeEdge is composed of the following components:
 
 - **Edged:** an agent that runs on edge nodes and manages containerized applications.
 - **EdgeHub:** a web socket client responsible for interacting with Cloud Service for the edge computing (like Edge Controller as in the KubeEdge Architecture). This includes syncing cloud-side resource updates to the edge, and reporting edge-side host and device status changes to the cloud.
+- **CloudHub:** a web socket server responsible for watching changes at the clouds side, cache and sending messages to EdgeHub.
```

Comment:
a web socket server responsible for watching changes at the clouds side, cache and sending messages to EdgeHub ->

A web socket server responsible for watching changes at the cloud side and sending messages to EdgeHub.

Reply:
Done, Thanks